### ISSUE TITLE
Church scene provided images and poses do not match

### ISSUE THREAD

**Issue by mli0603** [isl-org/TanksAndTemples#42]
Dear authors,

Thank you very much for the dataset! In the provided training scene "Church", the number of images does not match the number of poses (507 vs 644). Quite a few images are missing in the provided training set.

I tried to extract the images myself using the given timestamp file. However, the extracted images look different from the provided images even if they have the same index, and COLMAP doesn't give a meaningful reconstruction result. I also tried `ffmpeg` extraction specified in this link (https://www.tanksandtemples.org/tutorial/), but again the number of images (only 643 images) and poses do not match.

I wonder if you know what the issue is. Thank you very much!

**Comment by arknapit** [comment]
Yes, thats mysterious, I will need some time to search for the original data which was used for those reconstructions. Sorry for the inconvenience in the meanwhile.

greetings
Arno

**Comment by mli0603** [comment]
Thank you very much @arknapit! Look forward to the fixed dataset.

**Comment by mli0603** [comment]
Hi @arknapit, I wonder if you have any updates on this thread :smile:  Thank you!

**Comment by arknapit** [comment]
I am a bit busy at the time, so it will probably take another week or two until I finish it. In the meanwhile, I think I figured out what happened to the Church sequence: by mistake, I selected the undistorted COLMAP images folder (in the dense folder, after reconstruction), therefore, there are images missing, since not all of them made it to the reconstruction. The image ids should still match the *.log file ids though, so alignment and validation with your model/reconstruction should work.

Since I dont have the original COLMAP reconstruction raw files anymore, I can't correct for that. However, I think I can recover the original sampling timestamps from the video, so I might be able to provide you with the original images hopefully soon. (and will replace it on our download site). I keep you postet.

**Comment by mli0603** [comment]
Thank you for your reply! @arknapit

**Comment by arknapit** [comment]
I think I resampled the original image sequence:
https://drive.google.com/file/d/1yQcZTwC6ksb7oJ-d02oeKm67zim8xrX1/view?usp=sharing
I still have to check if they are also corresponding to the views from the *.log-file. Let me know in case you have tested them before me.

**Comment by mli0603** [comment]
Hi @arknapit 

I think the number of poses doesn't match the images. The log file has 644 poses but the number of images is 643.

**Comment by arknapit** [comment]
OK, I see. I will still need some time to find a solution here. In the worst case, I will just run COLMAP again, and put new log- and alignment files online. Sorry about the inconvenience in the menawhile.
